libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -195,6 +195,7 @@ class AP_Airspeed
         float	corrected_pressure;
         float   EAS2TAS;
         bool	healthy:1;
+        bool	prev_healthy:1;
```

Comment:
Do we need this in the state? In `AP_Airspeed::read()` we can cache the healthy state before calling get_pressure(), and use that on line 360? Otherwise looks good.